src/index.js

```diff
@@ -16,14 +16,18 @@ import {
 
 const threadPool = new HappyPack.ThreadPool({ size: os.cpus().length })
 
+
+// Add HappyPack to a block
+// Input: webpack block function -- (context, utils) => prevConfig => newConfig
+// Return new block function
 const happifyBlock = (
   block: WebpackBlock,
   { loaders, ...happypackOptions }: BlockOptions
-): WebpackBlock => (...args): Block => {
-  const compiledBlock = block(...args)
+): WebpackBlock => (context, utils): Block => prevConfig => {
+  const compiledBlock = block(context, utils)(prevConfig)
   const originalRules = getRules(compiledBlock)
 
-  if (!originalRules) return compiledBlock
+  if (!originalRules) return prevConfig
```

Comment:
Wasn't clear to me if this should return `prevConfig` or `compiledBlock` when there's nothing to change...

Reply:
When there's nothing to change, return the unchanged `prevConfig`. It's fine 👍